lib/pengine/complex.c

```diff
@@ -632,7 +631,7 @@ pe__unpack_resource(xmlNode *xml_obj, pe_resource_t **rsc,
     if ((*rsc)->variant == pe_unknown) {
         pe_err("Unknown resource type: %s", crm_element_name((*rsc)->xml));
         free(*rsc);
-        return FALSE;
+        return pcmk_rc_schema_validation;
```

Comment:
See above, under the case of `"Must specify id tag in <resource>"`. This isn't really a schema validation error but it fits the existing pattern.